back-end/www/models/model.py

```diff
@@ -42,3 +42,56 @@ class User(db.Model):
     def __repr__(self):
         return "<User id=%r created_at=%r client_id=%r client_type=%r>" % (
                 self.id, self.created_at, self.client_id, self.client_type)
+
+
+class Location(db.Model):
+    """
+    Class representing a location.
+
+    Attributes
+    ----------
+    id : int
+        Unique identifier as primary key.
+
+    factory_id : str
+        The uuid imported from disfactory factory table.
+    
+    year: integer
+        year Marks which year the satellite photo was taken from our geo sources. 
+
+    url : string
+        URL stores the location on the map.
+
+    bbox_left_up_lat
```

Comment:
please put commas between the variables and put them in one line, as shown in https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
please also document the data type